hugegraph-dist/src/assembly/static/conf/rest-server.properties

```diff
@@ -6,6 +6,8 @@ restserver.url=http://127.0.0.1:8080
 # graphs list with pair NAME:CONF_PATH
 graphs=[hugegraph:conf/hugegraph.properties]
 
+batch.max_write_ratio=80
```

Comment:
add comment: "The maximum thread ratio for batch writing, only take effect if the batch.max_write_threads is 0."
and add batch.max_write_threads=0

also update MAX_WRITE_THREADS description: 
`batch.max_write_ratio * total-rest-threads`
=>
`batch.max_write_ratio * restserver.max_worker_threads`